virtme/commands/run.py

```diff
@@ -968,6 +969,42 @@ def cleanup_console_script():
     qemuargs.extend(["-device", "vhost-vsock-pci,guest-cid=%d" % args.port])
 
 
+def ssh_client(args):
+    if args.remote_cmd is not None:
+        exec_escaped = args.remote_cmd.replace('"', '\\"')
+        remote_cmd = ["bash", "-c", exec_escaped]
+    else:
+        remote_cmd = []
+
+    cmd = ["ssh", "-p", str(args.port), "localhost"] + remote_cmd
+
+    if args.dry_run:
+        print(" ".join(cmd))
+    else:
+        os.execvp("ssh", cmd)
+
+
+def ssh_server(args, arch, qemuargs, kernelargs):
+    # Check if we need to generate the ssh host keys for the guest.
+    ssh_key_dir = f"{CACHE_DIR}/.ssh"
+    os.makedirs(f"{ssh_key_dir}/etc/ssh", exist_ok=True)
+    os.system(f"ssh-keygen -A -f {ssh_key_dir}")
+
+    # Implicitly enable dhcp to automatically get an IP on the network
+    # interface and prevent interface renaming.
+    kernelargs.extend(["virtme.dhcp", "net.ifnames=0", "biosdevname=0"])
+
+    # Tell virtme-ng-init / virtme-init to start sshd and use the current
+    # username keys/credentials.
+    username = get_username()
+    kernelargs.extend(["virtme.ssh"])
+    kernelargs.extend([f"virtme_ssh_user={username}"])
+
+    # Setup a port forward network interface for the guest.
+    qemuargs.extend(["-device", "%s,netdev=ssh" % (arch.virtio_dev_type("net"))])
+    qemuargs.extend(["-netdev", "user,id=ssh,hostfwd=tcp::%d-:22" % args.port])
```

Comment:
As mentioned on https://github.com/arighi/virtme-ng-init/pull/16#pullrequestreview-2526363654, this should be modified:

    "hostfwd=tcp:127.0.0.1:%d-:22" % args.port